includes/class-wc-admin-loader.php

```diff
@@ -145,6 +121,9 @@ private static function get_path( $file ) {
 	 * Class loader for enabled WooCommerce Admin features/sections.
 	 */
 	public static function load_features() {
+		require_once WC_ADMIN_ABSPATH . 'includes/page-controller/class-wc-admin-page-controller.php';
```

Comment:
Good call loading these here. I was worried about possible function name collision between the plugin here, and this shipping in core -- but loading here alongside the other feature classes should be late enough.